File: web/src/model/victron_controller/dashboard/from_0_2_0_cores-state.ts
// @ts-nocheck
import {CoresState as v0_2_0_CoresState} from './v0_2_0/CoresState'
import {CoresState as dashboard_CoresState} from './CoresState'

export function convert__cores_state__from__0_2_0(from: v0_2_0_CoresState): dashboard_CoresState {
    return new dashboard_CoresState (
        JSON.parse(JSON.stringify(from.cores)),
        from.topo_order
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_cores_state.rs


pub fn convert__cores_state__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::cores_state::CoresState) -> crate::victron_controller::dashboard::cores_state::CoresState {
    crate::victron_controller::dashboard::cores_state::CoresState {
        cores: serde_json::from_value(serde_json::to_value(&from.cores).unwrap()).unwrap(),
        topo_order: from.topo_order.clone(),
    }
}